src/components/buttons/secondary-button/secondary-button.spec.js

```diff
@@ -73,7 +73,7 @@ describe('rendering', () => {
   describe('when using "linkTo"', () => {
     it('should navigate to link when clicked', async () => {
       const { getByLabelText, history } = render(
-        <SecondaryButton {...props} linkTo="/foo/bar" />
+        <SecondaryButton {...props} onClick={null} linkTo="/foo/bar" />
```

Comment:
this test failed in same way as the current issues in MC-FE before making the code change 👍